CHANGELOG_PENDING.md

```diff
@@ -6,6 +6,8 @@
 
 #### General
 
+- \#1911 [Experimental] Enable content detection (@jailuthra,@yondonfu)
```

Comment:
Let's add a note that only adult/soccer is supported at the moment since we are only loading the model for the adult/soccer profile on node startup.